agent/README.md

```diff
@@ -34,3 +34,29 @@ Optional commands are parsed in the configuration module, but not implemented!
 ```
 cargo test
 ```
+
+## Logger
+### Usage
+The logger is configured via environment variables using [env_logger](https://docs.rs/env_logger/latest/env_logger/) crate. ```error!``` macro is the highest-priority and ```trace!``` the lowest, if no log level is provided if defaults to ```error```. [Filters](https://docs.rs/env_logger/latest/env_logger/#filtering-results) may be used to turn off the logging in a specific crate for example.
+
+```shell
+RUST_LOG=warn ./target/debug/tidybee-agent
```

Comment:
Windows equivalent ?

Reply:
@Bootoyka There is multiple ways to do that but I think this is not some something we will need to care about until we aren't clear on how the agent will be launched (at least on windows). We can't lose time on something this trivial for now

Reply:
```suggestion
RUST_LOG=warn ./target/debug/tidybee-agent

OR

pwsh -Command { $env:MYVAR="myvalue"; .\path\to.exe }
```